src/library/scala/collection/SortedSet.scala

```diff
@@ -26,8 +26,8 @@ trait SortedSet[A] extends Set[A] with SortedSetOps[A, SortedSet, SortedSet[A]]
   override protected[this] def writeReplace(): AnyRef = new DefaultSerializationProxy(sortedIterableFactory.evidenceIterableFactory[A], this)
 }
 
-trait SortedSetOps[A, +CC[X] <: SortedSet[X], +C <: SortedSetOps[A, CC, C]]
-  extends SetOps[A, Set, C]
+trait SortedSetOps[A, +CC[X] <: SortedSetOps[X, CC, _] with SortedSet[X], +C <: SortedSetOps[A, CC, C] with CC[A]]
+  extends InvariantSetOps[A, CC, C]
```

Comment:
Can `SortedSet` be made covariant too?

Reply:
No, because if I have a `SortedSet[Int]` I don't have a `SortedSet[Any]` since the latter would require an `Ordering[Any]` which is not something I can construct from an `Ordering[Int]`.

Reply:
Indeed!